app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Send me reset password instructions" %>
```

Comment:
I think it's better to keep it  _Send me reset password instructions_ instead of _submit_. That line makes it clear what it does. I have seen people using such lengthy lines like _send me the validation code_ in forgot password page. Some other ideas are  giving a line like "No worries. Enter your email we will send you the reset instructions" and email field and a button with _Send Request_ label.